en/doc_directory.md

```diff
@@ -0,0 +1,38 @@
+# Document Directory
+
+SCRF's Document Directory is a comprehensive guide of all SCRF documentation.
+
+## Awards, Bounties, Grants
+
+* [Research Summary Grants](en/grants_general_contributors.md)
+* [Pan-African Microgrants](en/grants_pan_african.md)
+* [Contributor Recognition and Rewards Initiative](en/grants_contributor_rewards.md)
+* [Impact Award](en/grants_impact_award.md)
+
+## Events
+
+* [Weekly Community Calls](en/content_community_calls)
+
+## Contributor Resources
+
+* [How to Get Compensated for Work](en/content_how_to_get_compensated_for_work.md)
+* [Research Improvement Process](en/process_research_improvement_proposal.md)
+* [Research Improvement Proposal](en/project_research_improvement_proposal.md)
+
+## Forum Templates
+
+* [Research Summary](en/content_research_summary_template.md)
+* [Discussion Posts](en/content_discussion_post_template.md)
+* [Mini-Posts](en/content_mini_post_template.md)
+* [Notable Works](en/content_notable_works_template.md)
+
+## Web3 Reference Materials
+
+* [Notable Works](en/content_notable_works.md)
+* [Terms Glossary](en/reference_terms_glossary.md)
+
+## SCRF Reference Materials
+
+* [Writing Style Guide](en/reference_style_guide.md)
+
+## Other
```

Comment:
Do we need a category if nothing is in it?

Reply:
Okay, I will remove it until we have items to populate it.